doc/stdlib/base.rst

```diff
@@ -664,6 +664,17 @@ Generic Functions
        julia> [1:5;] |> x->x.^2 |> sum |> inv
        0.01818181818181818
 
+.. function:: |>(xs::Tuple, f)
+
+   .. Docstring generated from Julia source
```

Comment:
I'm sorry, I was away for a while, and haven't catch up to recent changes. I just copied it over. Where do I need to put this .jl code then, for this to be true?